lib/notifications/display.dart

```diff
@@ -234,6 +234,13 @@ class NotificationDisplayManager {
     final groupKey = _groupKey(data.realmUrl, data.userId);
     final conversationKey = _conversationKey(data, groupKey);
 
+    final globalStore = await ZulipBinding.instance.getGlobalStore();
+    final account = globalStore.accounts.firstWhereOrNull((account) =>
+      account.realmUrl.origin == data.realmUrl.origin && account.userId == data.userId);
+    if (account == null) {
+      return;
+    }
```

Comment:
This deserves a code comment, explaining that it's about logged-out accounts where the unregister-token request failed for some reason.